tests/integration/sites_test.py

```diff
@@ -133,15 +97,15 @@ def test_rates(self, vasp_jumps):
         assert isclose(rates_std, 41893421993.683655)
 
     def test_activation_energies(self, vasp_jumps, vasp_sites):
-        activation_energies = vasp_jumps.activation_energies()
+        activation_energies = vasp_jumps.activation_energies(n_parts=10)
 
         assert isinstance(activation_energies, dict)
         assert len(activation_energies) == 1
 
         e_act, e_act_std = activation_energies[('48h', '48h')]
 
-        assert isclose(e_act, 0.1744591, abs_tol=1e-4)
-        assert isclose(e_act_std, .00405951, abs_tol=1e-6)
+        assert isclose(e_act, 0.20223, abs_tol=1e-4)
+        assert isclose(e_act_std, 0.00595, abs_tol=1e-6)
```

Comment:
Why are these values so different, do you have an explanation?

Reply:
So this problem arised because unoccupied sites where set to 1 instead of 0. Should be fixed now.